src/linux/Packaging.Linux/dotnet-tool.nuspec

```diff
@@ -0,0 +1,15 @@
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+    <metadata>
+        <id>GitCredentialManager.Cli</id>
+        <version>$version$</version>
+        <description>Secure, cross-platform Git credential storage with authentication to Azure Repos, GitHub, and other popular Git hosting services.</description>
+        <authors>The Git Fundamentals Team</authors>
```

Comment:
I would change this to `GitCredentialManager`.. the project exists separately from GitHub, and would then match the project org name.

Reply:
..or rather "Git Credential Manager" with spaces :)

Reply:
Sounds good. After re-reviewing the docs, I don't think `Company` is a required field either, so I'll go ahead and remove that.